2016/11/19 11:13:57 [error] 1078#1078: *29 FastCGI sent in stderr: "PHP message: PHP Warning:  require(/var/www/case/config/module_user.php): failed to open stream: No such file or directory in /var/www/case/config/web.php on line 52
PHP message: PHP Fatal error:  require(): Failed opening required '/var/www/case/config/module_user.php' (include_path='.:/usr/share/php') in /var/www/case/config/web.php on line 52" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/11/19 11:15:30 [error] 1078#1078: *31 FastCGI sent in stderr: "PHP message: PHP Warning:  require(/var/www/case/config/module_user.php): failed to open stream: No such file or directory in /var/www/case/config/web.php on line 52
PHP message: PHP Fatal error:  require(): Failed opening required '/var/www/case/config/module_user.php' (include_path='.:/usr/share/php') in /var/www/case/config/web.php on line 52" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/11/28 18:11:05 [error] 1056#1056: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /template/index.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/11/28 18:12:11 [error] 1056#1056: *8 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /template/index.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/11/28 18:12:12 [error] 1056#1056: *8 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /template/index.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/11/28 18:13:34 [error] 1056#1056: *11 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /t2/index.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/11/29 12:33:16 [error] 1100#1100: *28 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Method yii\bootstrap\ActiveField::__toString() must not throw an exception, caught Error: Class 'app\models\ArrayHelper' not found in /var/www/case/modules/cases/views/admin/_case.php on line 0" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /admin/cases/create HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local", referrer: "http://case.local/admin/cases/index"
2016/12/01 13:33:22 [error] 1026#1026: *804 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
ReflectionException: Method yii\widgets\BaseListView::yii\widgets\{closure}() does not exist in /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php:199
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(199): ReflectionMethod->__construct('yii\\widgets\\Bas...', 'yii\\widgets\\{cl...')
#1 /var/www/case/vendor/yiisoft/yii2/views/errorHandler/callStackItem.php(26): yii\web\ErrorHandler->addTypeLinks('yii\\widgets\\Bas...')
#2 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(249): require('/var/www/case/v...')
#3 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(307): yii\web\ErrorHandler->renderFile('@yii/views/erro...', Array)
#4 /var/www/case/vendor/yiisoft/yii2/views/errorHandler/exception.php(385): yii\web\ErrorHandler->renderCallStackItem(NULL, NULL, 'yii\\widgets\\Bas...', 'yii\\widgets\\{cl...', Array, 15)
#5 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(249): require('/var/www/case/v...')
#6 /var/" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /admin/operations/index HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local", referrer: "http://case.local/admin/operations/create"
2016/12/01 18:05:36 [error] 1028#1028: *1158 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
ReflectionException: Method yii\base\View::{closure}() does not exist in /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php:199
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(199): ReflectionMethod->__construct('yii\\base\\View', '{closure}')
#1 /var/www/case/vendor/yiisoft/yii2/views/errorHandler/callStackItem.php(26): yii\web\ErrorHandler->addTypeLinks('yii\\base\\View::...')
#2 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(249): require('/var/www/case/v...')
#3 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(307): yii\web\ErrorHandler->renderFile('@yii/views/erro...', Array)
#4 /var/www/case/vendor/yiisoft/yii2/views/errorHandler/exception.php(385): yii\web\ErrorHandler->renderCallStackItem(NULL, NULL, 'yii\\base\\View', '{closure}', Array, 3)
#5 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(249): require('/var/www/case/v...')
#6 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(113):" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /admin/operations/view?id=6 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local", referrer: "http://case.local/admin/operations/create"
2016/12/05 12:18:43 [error] 963#963: *1 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
PDOException: SQLSTATE[HY000] [1045] Access denied for user 'case'@'localhost' (using password: YES) in /var/www/case/vendor/yiisoft/yii2/db/Connection.php:602
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(602): PDO->__construct('mysql:host=loca...', 'case', 'z1Ec3U2udjihvms...', NULL)
#1 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(544): yii\db\Connection->createPdoInstance()
#2 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(863): yii\db\Connection->open()
#3 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(850): yii\db\Connection->getMasterPdo()
#4 /var/www/case/vendor/yiisoft/yii2/db/Command.php(219): yii\db\Connection->getSlavePdo()
#5 /var/www/case/vendor/yiisoft/yii2/db/Command.php(894): yii\db\Command->prepare(true)
#6 /var/www/case/vendor/yiisoft/yii2/db/Command.php(362): yii\db\Command->queryInternal('fetchAll', NULL)
#7 /var/www/case/vendor/yiisoft/yii2/db/mysql/Schema.php(199): yii\db\Command->quer" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /balance HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/05 12:20:56 [error] 964#964: *28 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
PDOException: SQLSTATE[HY000] [1044] Access denied for user 'case'@'localhost' to database 'case' in /var/www/case/vendor/yiisoft/yii2/db/Connection.php:602
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(602): PDO->__construct('mysql:host=loca...', 'case', 'z1Ec3U2udjihvms...', NULL)
#1 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(544): yii\db\Connection->createPdoInstance()
#2 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(863): yii\db\Connection->open()
#3 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(850): yii\db\Connection->getMasterPdo()
#4 /var/www/case/vendor/yiisoft/yii2/db/Command.php(219): yii\db\Connection->getSlavePdo()
#5 /var/www/case/vendor/yiisoft/yii2/db/Command.php(894): yii\db\Command->prepare(true)
#6 /var/www/case/vendor/yiisoft/yii2/db/Command.php(362): yii\db\Command->queryInternal('fetchAll', NULL)
#7 /var/www/case/vendor/yiisoft/yii2/db/mysql/Schema.php(199): yii\db\Command->queryAl" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /balance HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/05 12:20:57 [error] 964#964: *28 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
PDOException: SQLSTATE[HY000] [1044] Access denied for user 'case'@'localhost' to database 'case' in /var/www/case/vendor/yiisoft/yii2/db/Connection.php:602
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(602): PDO->__construct('mysql:host=loca...', 'case', 'z1Ec3U2udjihvms...', NULL)
#1 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(544): yii\db\Connection->createPdoInstance()
#2 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(863): yii\db\Connection->open()
#3 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(850): yii\db\Connection->getMasterPdo()
#4 /var/www/case/vendor/yiisoft/yii2/db/Command.php(219): yii\db\Connection->getSlavePdo()
#5 /var/www/case/vendor/yiisoft/yii2/db/Command.php(894): yii\db\Command->prepare(true)
#6 /var/www/case/vendor/yiisoft/yii2/db/Command.php(362): yii\db\Command->queryInternal('fetchAll', NULL)
#7 /var/www/case/vendor/yiisoft/yii2/db/mysql/Schema.php(199): yii\db\Command->queryAl" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /balance HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/05 12:20:57 [error] 964#964: *28 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
PDOException: SQLSTATE[HY000] [1044] Access denied for user 'case'@'localhost' to database 'case' in /var/www/case/vendor/yiisoft/yii2/db/Connection.php:602
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(602): PDO->__construct('mysql:host=loca...', 'case', 'z1Ec3U2udjihvms...', NULL)
#1 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(544): yii\db\Connection->createPdoInstance()
#2 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(863): yii\db\Connection->open()
#3 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(850): yii\db\Connection->getMasterPdo()
#4 /var/www/case/vendor/yiisoft/yii2/db/Command.php(219): yii\db\Connection->getSlavePdo()
#5 /var/www/case/vendor/yiisoft/yii2/db/Command.php(894): yii\db\Command->prepare(true)
#6 /var/www/case/vendor/yiisoft/yii2/db/Command.php(362): yii\db\Command->queryInternal('fetchAll', NULL)
#7 /var/www/case/vendor/yiisoft/yii2/db/mysql/Schema.php(199): yii\db\Command->queryAl" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /balance HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/05 12:20:57 [error] 964#964: *28 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
PDOException: SQLSTATE[HY000] [1044] Access denied for user 'case'@'localhost' to database 'case' in /var/www/case/vendor/yiisoft/yii2/db/Connection.php:602
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(602): PDO->__construct('mysql:host=loca...', 'case', 'z1Ec3U2udjihvms...', NULL)
#1 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(544): yii\db\Connection->createPdoInstance()
#2 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(863): yii\db\Connection->open()
#3 /var/www/case/vendor/yiisoft/yii2/db/Connection.php(850): yii\db\Connection->getMasterPdo()
#4 /var/www/case/vendor/yiisoft/yii2/db/Command.php(219): yii\db\Connection->getSlavePdo()
#5 /var/www/case/vendor/yiisoft/yii2/db/Command.php(894): yii\db\Command->prepare(true)
#6 /var/www/case/vendor/yiisoft/yii2/db/Command.php(362): yii\db\Command->queryInternal('fetchAll', NULL)
#7 /var/www/case/vendor/yiisoft/yii2/db/mysql/Schema.php(199): yii\db\Command->queryAl" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /balance HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/05 13:36:08 [error] 964#964: *115 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Class 'yarcode\freekassa\Merchant' not found in /var/www/case/config/web.php:61
Stack trace:
#0 /var/www/case/web/index.php(10): require()
#1 {main}
  thrown in /var/www/case/config/web.php on line 61" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/06 18:01:53 [error] 979#979: *134 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
ReflectionException: Method GuzzleHttp\Handler\Proxy::GuzzleHttp\Handler\{closure}() does not exist in /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php:199
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(199): ReflectionMethod->__construct('GuzzleHttp\\Hand...', 'GuzzleHttp\\Hand...')
#1 /var/www/case/vendor/yiisoft/yii2/views/errorHandler/callStackItem.php(26): yii\web\ErrorHandler->addTypeLinks('GuzzleHttp\\Hand...')
#2 /var/www/case/vendor/yiisoft/yii2/base/View.php(328): require('/var/www/case/v...')
#3 /var/www/case/vendor/yiisoft/yii2/base/View.php(250): yii\base\View->renderPhpFile('/var/www/case/v...', Array)
#4 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(253): yii\base\View->renderFile('/var/www/case/v...', Array, Object(yii\web\ErrorHandler))
#5 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(307): yii\web\ErrorHandler->renderFile('@yii/views/erro...', Array)
#6 /var/www/case/vendor/yiisof" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /operations/index/cashout HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/06 18:02:45 [error] 979#979: *134 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
ReflectionException: Method GuzzleHttp\Handler\Proxy::GuzzleHttp\Handler\{closure}() does not exist in /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php:199
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(199): ReflectionMethod->__construct('GuzzleHttp\\Hand...', 'GuzzleHttp\\Hand...')
#1 /var/www/case/vendor/yiisoft/yii2/views/errorHandler/callStackItem.php(26): yii\web\ErrorHandler->addTypeLinks('GuzzleHttp\\Hand...')
#2 /var/www/case/vendor/yiisoft/yii2/base/View.php(328): require('/var/www/case/v...')
#3 /var/www/case/vendor/yiisoft/yii2/base/View.php(250): yii\base\View->renderPhpFile('/var/www/case/v...', Array)
#4 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(253): yii\base\View->renderFile('/var/www/case/v...', Array, Object(yii\web\ErrorHandler))
#5 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(307): yii\web\ErrorHandler->renderFile('@yii/views/erro...', Array)
#6 /var/www/case/vendor/yiisof" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /operations/index/cashout HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/06 18:02:46 [error] 979#979: *134 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
ReflectionException: Method GuzzleHttp\Handler\Proxy::GuzzleHttp\Handler\{closure}() does not exist in /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php:199
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(199): ReflectionMethod->__construct('GuzzleHttp\\Hand...', 'GuzzleHttp\\Hand...')
#1 /var/www/case/vendor/yiisoft/yii2/views/errorHandler/callStackItem.php(26): yii\web\ErrorHandler->addTypeLinks('GuzzleHttp\\Hand...')
#2 /var/www/case/vendor/yiisoft/yii2/base/View.php(328): require('/var/www/case/v...')
#3 /var/www/case/vendor/yiisoft/yii2/base/View.php(250): yii\base\View->renderPhpFile('/var/www/case/v...', Array)
#4 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(253): yii\base\View->renderFile('/var/www/case/v...', Array, Object(yii\web\ErrorHandler))
#5 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(307): yii\web\ErrorHandler->renderFile('@yii/views/erro...', Array)
#6 /var/www/case/vendor/yiisof" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /operations/index/cashout HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/12 17:27:13 [error] 1012#1012: *257 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Class 'AccessControl' not found in /var/www/case/config/web.php:87
Stack trace:
#0 /var/www/case/web/index.php(10): require()
#1 {main}
  thrown in /var/www/case/config/web.php on line 87" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /settings/manager/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/12 17:27:26 [error] 1012#1012: *257 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to undefined function AccessControl() in /var/www/case/config/web.php:87
Stack trace:
#0 /var/www/case/web/index.php(10): require()
#1 {main}
  thrown in /var/www/case/config/web.php on line 87" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /settings/manager/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/12 17:30:20 [error] 1012#1012: *277 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Class 'AccessControl' not found in /var/www/case/config/web.php:88
Stack trace:
#0 /var/www/case/web/index.php(10): require()
#1 {main}
  thrown in /var/www/case/config/web.php on line 88" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /settings/manager/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/12 17:30:44 [error] 1012#1012: *277 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Class 'AccessRule' not found in /var/www/case/config/web.php:90
Stack trace:
#0 /var/www/case/web/index.php(10): require()
#1 {main}
  thrown in /var/www/case/config/web.php on line 90" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /settings/manager/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/12 17:34:36 [error] 1012#1012: *331 FastCGI sent in stderr: "PHP message: PHP Parse error:  syntax error, unexpected end of file, expecting ';' in /var/www/case/config/access.php on line 16" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /settings/manager/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local", referrer: "http://case.local/user/login"
2016/12/13 19:16:29 [error] 966#966: *98 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Cannot declare class app\models\RegistrationForm because the name is already in use in /var/www/case/models/RegistrationForm.php on line 7" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /user/register HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local", referrer: "http://case.local/user/login"
2016/12/16 15:18:13 [error] 963#963: *591 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
ReflectionException: Method yii\widgets\BaseListView::yii\widgets\{closure}() does not exist in /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php:199
Stack trace:
#0 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(199): ReflectionMethod->__construct('yii\\widgets\\Bas...', 'yii\\widgets\\{cl...')
#1 /var/www/case/vendor/yiisoft/yii2/views/errorHandler/callStackItem.php(26): yii\web\ErrorHandler->addTypeLinks('yii\\widgets\\Bas...')
#2 /var/www/case/vendor/yiisoft/yii2/base/View.php(328): require('/var/www/case/v...')
#3 /var/www/case/vendor/yiisoft/yii2/base/View.php(250): yii\base\View->renderPhpFile('/var/www/case/v...', Array)
#4 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(253): yii\base\View->renderFile('/var/www/case/v...', Array, Object(yii\web\ErrorHandler))
#5 /var/www/case/vendor/yiisoft/yii2/web/ErrorHandler.php(307): yii\web\ErrorHandler->renderFile('@yii/views/erro...', Array)
#6 /var/www/case/vendor/yiisoft/yi" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /operations/history/index?page=1 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/29 16:12:02 [error] 1073#1073: *294 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'morozovsk\websocket\Generic' not found in /var/www/case/vendor/morozovsk/websocket/Daemon.php on line 5" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /site/test2 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/29 16:14:06 [error] 1073#1073: *296 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'morozovsk\websocket\Generic' not found in /var/www/case/vendor/morozovsk/websocket/Daemon.php on line 5" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /site/test2 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/29 16:14:06 [error] 1073#1073: *296 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'morozovsk\websocket\Generic' not found in /var/www/case/vendor/morozovsk/websocket/Daemon.php on line 5" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /site/test2 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/29 16:14:07 [error] 1073#1073: *296 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'morozovsk\websocket\Generic' not found in /var/www/case/vendor/morozovsk/websocket/Daemon.php on line 5" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /site/test2 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/29 16:14:58 [error] 1073#1073: *296 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'morozovsk\websocket\Generic' not found in /var/www/case/vendor/morozovsk/websocket/Daemon.php on line 5" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /site/test2 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/29 19:07:50 [error] 1072#1072: *780 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /site/test2 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2016/12/30 19:43:38 [error] 1020#1020: *1 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Class 'app\websocket\Client' not found in /var/www/case/config/web.php:75
Stack trace:
#0 /var/www/case/web/index.php(10): require()
#1 {main}
  thrown in /var/www/case/config/web.php on line 75" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /site/test2 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local"
2017/01/04 13:56:43 [error] 1151#1151: *247 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'app\models\yii\base\Model' not found in /var/www/case/models/Refsys.php on line 7" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /cases/case/open?id=13&_pjax=%23p0 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local", referrer: "http://case.local/cases/case?id=13"
2017/01/04 13:56:43 [error] 1151#1151: *247 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'app\models\yii\base\Model' not found in /var/www/case/models/Refsys.php on line 7" while reading response header from upstream, client: 127.0.0.1, server: case.local, request: "GET /cases/case/open?id=13 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case.local", referrer: "http://case.local/cases/case?id=13"
2017/01/06 17:25:11 [error] 11900#0: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "case-opener.com"
2017/01/06 18:30:20 [error] 14149#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:30:20 [error] 14149#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:30:44 [error] 14149#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:30:44 [error] 14149#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:30:46 [error] 14149#0: *1 no live upstreams while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://localhost/ws/", host: "case-opener.com"
2017/01/06 18:30:47 [error] 14149#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:30:47 [error] 14149#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:30:47 [error] 14149#0: *1 no live upstreams while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://localhost/ws/", host: "case-opener.com"
2017/01/06 18:30:47 [error] 14149#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:30:47 [error] 14149#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:30:47 [error] 14149#0: *10 no live upstreams while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://localhost/ws/", host: "case-opener.com"
2017/01/06 18:30:47 [error] 14149#0: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:30:47 [error] 14149#0: *10 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:30:49 [error] 14149#0: *10 no live upstreams while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://localhost/ws/", host: "case-opener.com"
2017/01/06 18:33:30 [error] 14149#0: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:33:30 [error] 14149#0: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:34:30 [error] 14149#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:34:30 [error] 14149#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:35:30 [error] 14149#0: *41 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:35:30 [error] 14149#0: *41 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:36:30 [error] 14149#0: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:36:30 [error] 14149#0: *48 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:37:30 [error] 14149#0: *55 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:37:30 [error] 14149#0: *55 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:38:30 [error] 14149#0: *58 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:38:30 [error] 14149#0: *58 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:38:33 [error] 14149#0: *61 no live upstreams while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://localhost/ws/", host: "case-opener.com"
2017/01/06 18:39:12 [error] 14149#0: *63 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:39:12 [error] 14149#0: *63 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:39:22 [error] 14149#0: *63 no live upstreams while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://localhost/ws/", host: "case-opener.com"
2017/01/06 18:39:23 [error] 14149#0: *63 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:39:23 [error] 14149#0: *63 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:39:23 [error] 14149#0: *63 no live upstreams while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://localhost/ws/", host: "case-opener.com"
2017/01/06 18:39:23 [error] 14149#0: *63 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:39:23 [error] 14149#0: *63 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:39:30 [error] 14149#0: *72 no live upstreams while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://localhost/ws/", host: "case-opener.com"
2017/01/06 18:40:30 [error] 14149#0: *73 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:40:30 [error] 14149#0: *73 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:41:30 [error] 14149#0: *76 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:41:30 [error] 14149#0: *76 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:42:30 [error] 14149#0: *79 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:42:30 [error] 14149#0: *79 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/06 18:43:30 [error] 14149#0: *82 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://[::1]:8080/ws/", host: "case-opener.com"
2017/01/06 18:43:30 [error] 14149#0: *82 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8080/ws/", host: "case-opener.com"
2017/01/07 17:51:07 [error] 16224#0: *122 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 181.143.8.34, server: case-opener.com, request: "GET /admin/config.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "77.221.144.12"
2017/01/08 02:32:19 [error] 16224#0: *153 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 185.40.4.14, server: case-opener.com, request: "GET /recordings/index.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "77.221.144.12"
2017/01/08 23:43:00 [error] 16224#0: *271 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 185.40.4.14, server: case-opener.com, request: "GET /recordings/index.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "77.221.144.12"
2017/01/09 12:52:31 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:52:39 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:52:40 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:52:41 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:52:41 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:52:41 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:52:41 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:52:44 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:52:48 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:52:53 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:52:56 [error] 7297#0: *2 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 12:54:32 [error] 7297#0: *29 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:06:29 [error] 7698#0: *13 upstream prematurely closed connection while reading response header from upstream, client: 10.129.198.67, server: case-opener.com, request: "GET /ws/ HTTP/1.0", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:06:50 [error] 7698#0: *16 upstream prematurely closed connection while reading response header from upstream, client: 10.129.198.67, server: case-opener.com, request: "GET /ws/ HTTP/1.0", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:07:03 [error] 7698#0: *18 upstream prematurely closed connection while reading response header from upstream, client: 10.129.198.67, server: case-opener.com, request: "GET /ws/ HTTP/1.0", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:10:57 [error] 7698#0: *32 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:14:54 [error] 7698#0: *49 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:15:04 [error] 7698#0: *49 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:15:05 [error] 7698#0: *49 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:15:05 [error] 7698#0: *49 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:15:05 [error] 7698#0: *49 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:15:05 [error] 7698#0: *49 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:15:05 [error] 7698#0: *49 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:15:11 [error] 7698#0: *49 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:15:26 [error] 7698#0: *49 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:17:22 [error] 7698#0: *64 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/ws/", host: "case-opener.com"
2017/01/09 13:18:34 [error] 7951#0: *3 upstream prematurely closed connection while reading response header from upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:21:22 [error] 7951#0: *22 recv() failed (104: Connection reset by peer) while proxying upgraded connection, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:21:23 [error] 7951#0: *24 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:21:24 [error] 7951#0: *26 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:21:25 [error] 7951#0: *28 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:21:27 [error] 7951#0: *30 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:21:29 [error] 7951#0: *32 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:21:33 [error] 7951#0: *34 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:21:38 [error] 7951#0: *36 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:21:46 [error] 7951#0: *38 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:21:58 [error] 7951#0: *40 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:22:15 [error] 8027#0: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:22:41 [error] 8027#0: *15 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:23:20 [error] 8027#0: *17 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:24:18 [error] 8027#0: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:27:54 [error] 8027#0: *69 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:27:55 [error] 8027#0: *71 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
2017/01/09 13:27:56 [error] 8027#0: *73 connect() failed (111: Connection refused) while connecting to upstream, client: 91.207.98.73, server: case-opener.com, request: "GET /ws/ HTTP/1.1", upstream: "http://127.0.0.1:8004/", host: "case-opener.com"
